### DISCUSSION TITLE
Wrong nodejs version in latest images

### DISCUSSION THREAD

**Discussion by romulorm** [requarks/wiki#7183] [Error / Bug Report] [answered]
If I'm not wrong, nodejs version in latest docker images (2.5.303) must be 22, not 18. I think that's the reason for multiple errors in application (An unexpected error occurred).

**Comment by NGPixel** [comment, marked as answer]
No (Node.js 18 is still actively maintained/patched) and the errors you're getting are unrelated.

**Reply by romulorm** [reply]
I've installed in two machines, Ubuntu and Windows, using this docker-compose https://docs.requarks.io/install/docker#using-docker-compose.
